routes/users.js

```diff
@@ -11,6 +11,7 @@ const {redisClient} = require('../config/redis');
 const {validate} = require('../middlewares/joi-validation/validate');
 const {UserValidation} = require('../joi-validations/users.validations');
 const RegisterV2WithoutUploadPhotoMiddleware = require('../middlewares/register-user/registerV2WithoutUploadPhoto');
+const getTargetUserIdByPMChannelIdMiddleware = require('../middlewares/user/getTargetUserIdByPMChannelId');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new middleware `getTargetUserIdByPMChannelIdMiddleware` is imported correctly. However, it's important to ensure that this middleware function handles all possible error scenarios and passes them to the next middleware in the chain using `next(err)`. This way, your express application can handle these errors appropriately.
```diff
+ const getTargetUserIdByPMChannelIdMiddleware = require('../middlewares/user/getTargetUserIdByPMChannelId');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->